packages/core/admin/server/controllers/admin.js

```diff
@@ -96,11 +98,24 @@ module.exports = {
     );
     const isHostedOnStrapiCloud = env('STRAPI_HOSTING', null) === 'strapi.cloud';
 
+    const numberOfAllContentTypes = _.size(strapi.contentTypes);
+    const numberOfComponents = _.size(strapi.components);
+
+    const getNumberOfDynamicZones = () =>
+      pipe(
+        map('attributes'),
+        flatMap(values),
+        sumBy(propEq('type', 'dynamiczone'))
+      )(strapi.contentTypes);
+
     return {
       data: {
         useTypescriptOnServer,
         useTypescriptOnAdmin,
         isHostedOnStrapiCloud,
+        numberOfAllContentTypes, // TODO: Rename this in Strapi v5
```

Comment:
Would you mind adding a bit more context in case someone else should handle this in V5?
Maybe a short sentence about what it should be renamed to & why?

example:
(it could probably be phrased better though)
`// TODO: V5: This event should be renamed numberOfContentTypes in V5 as the name is already taken to describe the number of content types using i18n.`